### ISSUE TITLE
Allow passing the serializer to `serialize` helper method.

### ISSUE THREAD

**Issue by Burgestrand** [elabs/serial#5]
It's a small thing, but it makes for more consistent code in the controllers if the serializer needs to be changed.

**Comment by Burgestrand** [comment]
Closed with 6c9d8e799eef3e9da2adf34511fcafeceb7fe2e2.

**Comment by Burgestrand** [comment]
A cool side-effect of this is that you can *ALSO* pass an on-the-fly serializer, similar to this:

``` ruby
serialize(Person.first!) do |h, person|
  h.attribute(:name, person.name)
end
```

I'm not sure what I think of this as a public API, so it remains an undocumented feature.
